packages/shared/src/libs/enums/enums.ts

```diff
@@ -1,4 +1,5 @@
 export { APIPath } from "./api-path.enum.js";
 export { AppEnvironment } from "./app-environment.enum.js";
+export { AuthErrorMessages } from "./auth-error-messages.js";
```

Comment:
```suggestion
export { AuthErrorMessages } from "./auth-error-messages.enum.js";
```
follow the style of the application